packages/mobile/src/components/lineup-tile/CollectionTileTrackList.tsx

```diff
@@ -72,10 +79,11 @@ type TrackItemProps = {
   track?: LineupTrack
   uid?: UID
   isAlbum?: boolean
+  deleted: boolean
```

Comment:
Making deleted required is a bit odd imo. Optional defaulting falsy seems more appropriate imho